src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -1103,20 +1111,7 @@ export class EditorHistoryEntry extends EditorQuickOpenEntry {
 	}
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
-		if (mode === Mode.OPEN) {
-			const sideBySide = !context.quickNavigateConfiguration && context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
-			const pinned = !this.configurationService.getConfiguration<IWorkbenchEditorConfiguration>().workbench.editor.enablePreviewFromQuickOpen;
-
-			if (this.input instanceof EditorInput) {
-				this.editorService.openEditor(this.input, { pinned }, sideBySide).done(null, errors.onUnexpectedError);
-			} else {
-				this.editorService.openEditor({ resource: (this.input as IResourceInput).resource, options: { pinned: true } }, sideBySide);
-			}
-
-			return true;
-		}
-
-		return false;
+		return super.run(mode, context);
```

Comment:
@wprater `EditorQuickOpenEntry` has some behaviour that we do not want in `EditorPickerEntry` (e.g. opening to the side) but we do want the preview especially in the `EditorQuickOpenEntry` as well. I fear for now we have to duplicate this if it is not too much code...